packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -606,15 +612,27 @@ export default class Meetings extends WebexPlugin {
      * @memberof Meetings
      */
     syncMeetings() {
-      return this.request.getActiveMeetings().then((locus) => {
-        if (locus.loci && locus.loci.length > 0) {
-          locus.loci.forEach((locus) => {
+      return this.request.getActiveMeetings().then((locusArray) => {
+        const activeLocusUrl = [];
+
+        if (locusArray.loci && locusArray.loci.length > 0) {
```

Comment:
I'm not sure what this check is doing. It looks like if you have active meetings you push the new meetings into the `activeLocusUrl` array. Otherwise you iterate through the existing meetings and only delete them if they aren't in the `activeLocusUrl` array? Won't that always be empty since you set it to an empty array on 616?

Reply:
you are correct if there is active meeting i do push in ` activeLocusUrl.push(locus.url);` and then do a check . But there should not be a else condition removed it